nextflow_schema.json

```diff
@@ -42,8 +40,1026 @@
+                    "description": "Clean the result annotations to standardise them to Genbank/ENA conventions.",
+                    "help_text": "The resulting annotations are cleaned up to standardise them to Genbank/ENA/DDJB conventions. CDS without any attributed hits and those without gene symbols or product descriptions different from hypothetical will be marked as 'hypothetical'.\nWhen activated the '--min-contig-length' will be set to 200. More info can be found [here](https://github.com/oschwengers/bakta).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--compliant`\n",
+                    "fa_icon": "fas fa-check"
+                },
+                "annotation_bakta_skiptrna": {
+                    "type": "boolean",
+                    "description": "Skip tRNA detection & annotation.",
+                    "help_text": "This flag skips [tRNAscan-SE 2.0](http://lowelab.ucsc.edu/tRNAscan-SE/) that predicts tRNA genes. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--skip-trna`",
+                    "fa_icon": "fas fa-forward"
+                },
+                "annotation_bakta_skiptmrna": {
+                    "type": "boolean",
+                    "description": "Skip tmRNA detection & annotation.",
+                    "help_text": "This flag skips [Aragorn](http://www.ansikte.se/ARAGORN/) that predicts tmrna genes. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--skip-tmrna`\n`",
+                    "fa_icon": "fas fa-forward"
+                },
+                "annotation_bakta_skiprrna": {
+                    "type": "boolean",
+                    "description": "Skip rRNA detection & annotation.",
+                    "help_text": "This flag skips [Infernal vs. Rfam rRNA covariance models](http://eddylab.org/infernal/) that predicts rrna genes. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--rrna`",
+                    "fa_icon": "fas fa-forward"
+                },
+                "annotation_bakta_skipncrna": {
+                    "type": "boolean",
+                    "description": "Skip ncRNA detection & annotation.",
+                    "help_text": "This flag skips [Infernal vs. Rfam ncRNA covariance models](http://eddylab.org/infernal/) that predicts ncrna genes.\nBAKTA distinguishes between ncRNA genes and (cis-regulatory) regions to enable the distinction of feature overlap detection.\nThis including distinguishing between ncrna gene types: sRNA, antisense, ribozyme and antitoxin. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--ncrna`",
```

Comment:
```suggestion
                    "help_text": "This flag skips [Infernal vs. Rfam ncRNA covariance models](http://eddylab.org/infernal/) that predicts ncRNA genes.\nBAKTA distinguishes between ncRNA genes and (cis-regulatory) regions to enable the distinction of feature overlap detection.\nThis including distinguishing between ncrna gene types: sRNA, antisense, ribozyme and antitoxin. More details can be found in the [documentation](https://github.com/oschwengers/bakta/blob/main/README.md#usage).\n\n>  Modifies tool parameter(s):\n> - BAKTA: `--ncrna`",
```

Reply:
(Uppercased one more ncrna in line 168 in the review branch)